namespaces/live.cloud-platform.service.justice.gov.uk/data-platform-app-pfg-dash-moj-dev/00-namespace.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
     cloud-platform.justice.gov.uk/is-production: "false"
     cloud-platform.justice.gov.uk/environment-name: "dev"
   annotations:
-    cloud-platform.justice.gov.uk/business-unit: "Platforms"
-    cloud-platform.justice.gov.uk/slack-channel: "data-platform-application-migration-support"
+    cloud-platform.justice.gov.uk/business-unit: "HQ"
+    cloud-platform.justice.gov.uk/slack-channel: ""
     cloud-platform.justice.gov.uk/application: "pfg-dash-moj"
-    cloud-platform.justice.gov.uk/owner: "analytical-platform: analytics-platform-tech@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/owner: "People Analytics: peopleanalytics@justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/pfg-dash-moj"
-    cloud-platform.justice.gov.uk/team-name: "analytics-hq"
+    cloud-platform.justice.gov.uk/team-name: ""
```

Comment:
Details missing from migration [spreadsheet](https://docs.google.com/spreadsheets/d/1plYjK7UXCQkbTjgdhcZy-8qatdCLMFKaA2qgxHBzoaM/edit#gid=812827045)

I had assumed it should be `people-analytics` but this is missing form the [github](https://github.com/orgs/ministryofjustice/teams?query=people-analytics) organisations, so have removed for now